exonum/src/storage/proof_list_index/mod.rs

```diff
@@ -347,12 +346,9 @@ where
     /// ```
     pub fn get_proof(&self, index: u64) -> ListProof<V> {
         if index >= self.len() {
-            panic!(
-                "Index out of bounds: the len is {} but the index is {}",
-                self.len(),
-                index
-            );
+            return ListProof::Absent(index, self.list_hash());
```

Comment:
I believe we should document this behavior in rustdocs and maybe even mention in changelog. This is a new feature and users should be aware then they can use `get_proof()` with any index to get a proof of absence as well.